src/index.ts

```diff
@@ -1,7 +1,9 @@
+import { readFile } from 'fs';
 import { Plugin } from 'vite';
 import execa from 'execa';
 import npmRunPath from 'npm-run-path';
 import chalk from 'chalk';
+import parseErrorLog from './overlay';
```

Comment:
I am wondering why the filename is `overlay.ts` not something like `parseCompilerLog.ts`?

Latter seems more intuitive to me.

Reply:
6c04d59